build.gradle

```diff
@@ -138,18 +138,24 @@ project(':iceberg-data') {
 }
 
 project(':iceberg-hive') {
-    dependencies {
-      compile project(':iceberg-core')
+  dependencies {
+    compile project(':iceberg-core')
 
-      compileOnly "org.apache.hive:hive-standalone-metastore:$hiveVersion"
+    compileOnly "org.apache.avro:avro:$avroVersion"
```

Comment:
Why was it necessary to add Avro? Did this add a use?

Reply:
This line is needed to use `iceberg.DataFiles`. Otherwise, we get an exception:

```
java.lang.NoSuchMethodError: org.apache.avro.Schema.setLogicalType(Lorg/apache/avro/LogicalType;)V

	at org.apache.avro.LogicalType.addToSchema(LogicalType.java:72)
	at com.netflix.iceberg.avro.TypeToSchema.<clinit>(TypeToSchema.java:42)
	at com.netflix.iceberg.avro.AvroSchemaUtil.convert(AvroSchemaUtil.java:69)
	at com.netflix.iceberg.avro.AvroSchemaUtil.convert(AvroSchemaUtil.java:65)
	at com.netflix.iceberg.PartitionData.getSchema(PartitionData.java:38)
	at com.netflix.iceberg.PartitionData.<init>(PartitionData.java:67)
	at com.netflix.iceberg.DataFiles.newPartitionData(DataFiles.java:37)
	at com.netflix.iceberg.DataFiles.access$000(DataFiles.java:32)
	at com.netflix.iceberg.DataFiles$Builder.<init>(DataFiles.java:197)
	at com.netflix.iceberg.DataFiles.builder(DataFiles.java:165)
	at com.netflix.iceberg.hive.HiveTablesTest.testConcurrentFastAppends(HiveTablesTest.java:105)
...
```

I believe this happens because `iceberg-core` depends on Avro 1.8.2 but `hive-exec:3.1.0` will reference Avro 1.7.7 via `hadoop-common`. If I comment out this line, `avro.Schema` will come from `hive-exec`.

```
Class<Schema> klass = Schema.class;
klass.getResource('/' + klass.getName().replace('.', '/') + ".class");
// jar:file:/.../.gradle/caches/modules-2/files-2.1/org.apache.hive/hive-exec/3.1.0/1c4a955ecc53f6f486c2eb049bc38c02af239bfa/hive-exec-3.1.0.jar!/org/apache/avro/Schema.class
```

Reply:
That sounds fine to me. I just wanted to make sure this was necessary.